install_config/topics/configuring_gce_objects.adoc

```diff
@@ -0,0 +1,176 @@
+example, {product-title} node hosts can call the GCP disk API to provide a
+persistent volume to an application.
+
+Access control to the various infrastructure, service resources, and
+fine-grained roles are available using the IAM service. For more information, see link:https://cloud.google.com/compute/docs/access/[the Access cloud overview section of the GCP documentation].
+
+SSH keys::
+
+GCP injects SSH public keys as authorized keys to be able to log in using SSH in
+the created instances. The list of SSH keys can be configured per instance or
+per project.
+
+You can use existing SSH keys. GCP metadata can help store the SSH keys that are
+injected at boot time in the instances to allow SSH access.
+
+See link:https://cloud.google.com/compute/docs/storing-retrieving-metadata[the Metadata section of the GCP documentation] for more information. 
+
+GCP regions and zones::
+
+GCP has a global infrastructure that covers regions and availability
+zones. While deploying {product-title} in GCP on different zones can help avoid
+single-point-of-failures, there are some caveats regarding storage. 
+
+GCP disks are created within a zone. Therefore, if a {product-title} node host
+goes down in zone "A" and the pods should be moved to zone "B", the persistent
+storage cannot be attached to those pods because the disks are in a different
+zone.
+
+Deploying a single zone of multizone {product-title} environment is an important
+decision to make before installing {product-title}. If deploying a multizone
+environment, the recommended setup is to use three different zones in a single region.
+
+See link:https://cloud.google.com/compute/docs/zones[the GCP documentation on regions and zones] or link:https://cloud.google.com/kubernetes-engine/docs/concepts/regional-clusters[the Kubernetes documentation on multiple zones] for more information.
+
+External IP address::
+
+So that GCP instances can communicate with the Internet, you must attach an
+external IP address to the instance. Also, an external IP address is required to
+communicate with instances deployed in GCP from outside the VPC Network.
+
+[WARNING]
+====
+https://cloud.google.com/vpc/docs/vpc#internet_access_reqs[Requiring an `External IP address` for internet access] is a limitation of
+the provider. This section configures firewall rules to avoid
+incoming external traffic in instances if not needed.
+====
+
+See link:https://cloud.google.com/compute/docs/ip-addresses/[the GCP
+documentation on external IP address] for more information.
+
+Cloud DNS::
+
+GCP cloud DNS is a DNS service used to publish domain names to the global
+DNS using GCP DNS servers. 
+
+The Public cloud DNS zone requires a domain name either purchased through
+Google’s "Domains" service or through a third-party provider. When you create the zone, you must add the name servers provided by Google to the registrar.
+
+See link:https://cloud.google.com/dns/overview[the GCP documentation on Cloud DNS] for more information.
```

Comment:
Does this link include instructions for updating the registrar?

Reply:
They basically explain it here https://cloud.google.com/dns/update-name-servers#change_your_domain_registrars_name_servers_for_your_domain (well... 'explain'...)